module/documents/item.mjs

```diff
@@ -783,6 +783,7 @@ export default class Item5e extends Item {
         item.prepareFinalAttributes();
       }
     }
+    if ( isSpell ) foundry.utils.setProperty(options.flags, "dnd5e.use", {spellLevel: item.system.level, itemId: this.id});
```

Comment:
Huh, is the original item ID not already attached to the chat card?

Reply:
On `item.use()` there are no system flags at all. The item id and the spell level have always been present in the dataset, however. But as the issue says, this makes it easier.